inc/mailcollector.class.php

```diff
@@ -929,7 +929,7 @@ function buildTicket($uid, \Laminas\Mail\Storage\Message $message, $options = []
       // max size = 0 : no import attachments
       if ($this->fields['filesize_max'] > 0) {
          if (is_writable(GLPI_TMP_DIR)) {
-            $tkt['_filename'] = $this->getAttached($message, GLPI_TMP_DIR."/", $this->fields['filesize_max']);
+            $tkt['_content'] = $this->getAttached($message, GLPI_TMP_DIR."/", $this->fields['filesize_max']);
```

Comment:
I may be wrong, but inline attachement (images in a rich text) only should be pushed in `_content`. For attached documents (a PDF file for instance), should it be pushed in `_filename` ?

Reply:
I notice that the base branch has a bug here : inline images generate an incorrect img tag with "denied:" in the src attribute. I cancel this change which has no effect. The bug should be fixed with an other PR.